plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/ObjectIdFunctions.java

```diff
@@ -60,6 +63,15 @@ public static long objectidTimestamp(@SqlType("ObjectId") Slice value)
         return packDateTimeWithZone(SECONDS.toMillis(epochSeconds), UTC_KEY);
     }
 
+    @Description("Mongodb ObjectId from the given timestamp")
+    @ScalarFunction
+    @SqlType("ObjectId")
+    public static Slice timestampObjectid(@SqlType("timestamp(3) with time zone") long value)
```

Comment:
This should probably be timestamp(0) since the function doesn’t use the fractional seconds.

Reply:
Thank you! I was not aware of the meaning, I just took the type from the other functions in the class.
Updated it to be timestamp(0)